### ISSUE TITLE
[2019] Passage Re-ranking with BERT

### ISSUE THREAD

**Issue by rejasupotaro** [rejasupotaro/paper-reading#112]
## Summary

- A model on the leaderboard: [MS MARCO - Leaderboard](http://www.msmarco.org/leaders.aspx)

## link

https://arxiv.org/pdf/1901.04085.pdf

## Notes

Simply,

> We start training from a pre-trained BERT model and fine-tune it to our re-ranking task using the
cross-entropy loss:

<img width="360" alt="Screenshot 2020-01-07 at 11 20 14" src="https://user-images.githubusercontent.com/883148/71891963-b35ae280-313f-11ea-97a2-3478b053291d.png">

## BibTeX

```
@article{nogueira2019passage,
  title={Passage Re-ranking with BERT},
  author={Nogueira, Rodrigo and Cho, Kyunghyun},
  journal={arXiv preprint arXiv:1901.04085},
  year={2019}
}
```
